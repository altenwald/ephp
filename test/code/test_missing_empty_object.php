<?php

function empty_check() {
    global $object;
    if (!empty($object->error)) {
        print "isset check: NOOO!!!\n";
    } else {
        print "isset check: OK\n";
    }
}

function create() {
    global $object;
    $object = new stdClass;
}

create();
empty_check();
var_dump($object);
