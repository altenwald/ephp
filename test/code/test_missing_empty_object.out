
Notice: Undefined index: error in {{CWD}}/test/code/test_missing_empty_object.php on line 5
isset check: OK
object(stdClass)#1 (0) {
}
